scan-command/src/main/java/io/ballerina/scan/PlatformPluginContext.java

```diff
@@ -0,0 +1,45 @@
+/*
+ *
+ *  * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

Comment:
```suggestion
/*
 *  Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

Fix wherever applicable.